ci_framework/roles/ci_nmstate/filter_plugins/ci_nmstate_map_instance_nets_filter.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+  name: ci_nmstate_map_instance_nets
+  short_description: Maps CI-framework instance networks dict to a proper nmstate list of interfaces.
+  description:
+    - Maps CI-framework instance networks dict to a proper nmstate list of interfaces.
+  options:
+    _input:
+      description:
+        - CI-framework networks dictionary for a given instance.
+        - Each key represents a network.
+        - Each value represents properties of the interface attached to that network.
+      type: dict
+      required: true
+'''
+
+EXAMPLES = '''
+- name: Define data to work on in the examples below
+  vars:
+    instace_nets:
```

Comment:
Sorry I am not sure It is a typo. We should use the word `instance` instead of `instace`.